pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -449,7 +449,8 @@ func (c *Cluster) determineExtraMonToRemove() string {
 	arbitraryMon := ""
 	for _, mon := range mons {
 		if mon.NodeName == "" {
-			logger.Debugf("mon %q is not scheduled to a specific host", mon.DaemonName)
+			logger.Infof("mon %q is not scheduled to a specific host, removing portable mon %q", mon.DaemonName, mon.DaemonName)
```

Comment:
I'd suggest leave the debug statement as it was, and no need for this info message. The message below on line 467 will log which mon is actually being removed. So the only change we need for this PR is line 453.

Reply:
done